prepare/cards/tablerow_classify.py

```diff
@@ -0,0 +1,54 @@
+from src.unitxt.blocks import (
+    AddFields,
+    LoadFromKaggle,
+    MapInstanceValues,
+    RenameFields,
+    SerializeTableRowAsText,
+    SplitRandomMix,
+    TaskCard,
+)
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.operators import ExtractFieldValues
+from src.unitxt.test_utils.card import test_card
+
+card = TaskCard(
+    loader=LoadFromKaggle(
+        url="https://www.kaggle.com/datasets/fedesoriano/heart-failure-prediction"
+    ),
+    preprocess_steps=[
+        SplitRandomMix(
+            {"train": "train[70%]", "validation": "train[10%]", "test": "train[20%]"}
+        ),
+        RenameFields(field_to_field={"HeartDisease": "label"}),
+        MapInstanceValues(mappers={"label": {"0": "Normal", "1": "Heart Disease"}}),
+        AddFields(
+            fields={
+                "text_type": "Person",
```

Comment:
```suggestion
                "text_type": "person medical record",
```
I think it is more descrptive, no?

Reply:
Done